cmd/config_test.go

```diff
@@ -146,289 +244,5 @@ var _ = Describe("Config", func() {
 				Eventually(session).Should(clitest.ShouldFail())
 			})
 		})
-
-		Describe("validating configs", func() {
```

Comment:
`validating configs` and `validating configs with pipeline parameters` have been moved to the cucumber tests as they are slightly easier to reason with.